packages/inertia-vue3/src/useForm.js

```diff
@@ -146,9 +135,14 @@ export default function form(data = {}) {
     delete(url, options) {
       this.submit('delete', url, options)
     },
+  })
+
+  if (remember) {
+    watch(form, (value) => {
+      const raw = toRaw(unref(value))
+      Inertia.remember({ data: raw.data(), errors: raw.errors }, key)
```

Comment:
```suggestion
      const keepRememberedKeys = obj => {
        Object
          .keys(obj)
          .reduce((carry, field) => ({
            ...carry,
            ...(fields.length > 0 && !remember.includes(field) ? {} : { [field] : obj[field] }),
          }), {})
      }

      Inertia.remember({
        data: keepRememberedKeys(raw.data()),
        errors: keepRememberedKeys(raw.errors)
      }, key)
```
Would something like this be beneficial? Don't know if it does work, but the idea is you can also pass `remember: ['email']` to the form, all other form fields won't be remembered

Reply:
So I don't think I want users to have to specify which fields they want to remember. Generally speaking remembering the whole form is fine. Not to say we couldn't add more granularity like this in the future. I just think I want to start with a simple enable/disable option.